contracts/views.py

```diff
@@ -0,0 +1,208 @@
+from django.contrib import messages
```

Comment:
You can split views into separate files for better organization, for example
1. Move contract-related views to contracts/contract_views.py
2. Move tenant-related views to contracts/tenant_views.py